src/routing/PrivateRoute.tsx

```diff
@@ -1,14 +1,30 @@
 import React from 'react';
 
-import { Redirect, Route } from 'react-router-dom';
+import { Redirect, Route, RouteComponentProps } from 'react-router-dom';
 
 import { ROUTE_PATHS } from '@config';
 import { useAccounts } from '@services/Store';
+import { translateRaw } from '@translations';
+import { IAppRoute } from '@types';
+import { useEffectOnce } from '@vendor';
 
-interface PrivateRouteProps {
-  [key: string]: any;
+interface PrivateRouteProps extends IAppRoute {
+  key: number;
 }
 
+interface PageTitleProps extends RouteComponentProps {
+  pageComponent: any;
+  title?: string;
+}
+
+const PageTitleRoute = ({ pageComponent: Component, title, ...props }: PageTitleProps) => {
```

Comment:
have you tried with `react-helmet` ? https://dev.to/antibland/comment/140ag

Reply:
I didn't do it that way because I noticed that we didn't already have it as a dependency and figured it'd be better not to add it for something so small.

Reply:
I would prefer using a documented dependency to remove the burden of tests from our code.
But it's your call.